public/gcpv2_merged.json

```diff
@@ -1,100 +0,0 @@
-{
```

Comment:
Do we want to delete these? I think @mikaeld configured stuff at the workweek so that the default dev setup uses these but merges to main push assets except for these.

Reply:
As the project is currently configured, we need to keep those here.